app/views/webhooks/_message_status.njk

```diff
@@ -0,0 +1,16 @@
+{% set statusStyleMap = {
+"PENDING": "govuk-tag--grey",
+"SUCCESSFUL": "govuk-tag--green",
+"FAILED": "govuk-tag--red",
+"WILL_NOT_SEND": "govuk-tag--yellow"
+} %}
+{% set statusTextMap = {
+"PENDING": "PENDING",
+"SUCCESSFUL": "SUCCESSFUL",
+"FAILED": "FAILED",
+"WILL_NOT_SEND": "WON’T SEND"
```

Comment:
May as well make the label WILL NOT SEND because [we shouldn’t use negative contractions](https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#contractions).